boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -44,7 +44,7 @@
 	leds {
 		compatible = "gpio-leds";
 		green_led: led-1 {
-			gpios = <&gpio1 9 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio1 9 0>;
```

Comment:
0? Why not 1? Same applies to below instances.

Reply:
How do you see ACTIVE_HIGH/LOW working for LEDs?

Would it mean that for a LED (output), if configured to ACTIVE_HIGH, than writing '1' to the GPIO would turn on the LED, and '0' off.  If configured to ACTIVE_LOW, than writing '0' to the LED would turn it on, and writing '1' would turn it off.

Trying to see if we can remove GPIO_POL_NORMAL/GPIO_POL_INV.

Reply:
> Would it mean that for a LED (output), if configured to ACTIVE_HIGH, than writing '1' to the GPIO would turn on the LED, and '0' off. If configured to ACTIVE_LOW, than writing '0' to the LED would turn it on, and writing '1' would turn it off.

yep, that's how it works